docs/document/website/translated_docs/ja/version-1.10.0/components/desktop/dialog.md

```diff
@@ -0,0 +1,120 @@
+---
+id: version-1.10.0-dialog
+title: Dialog
+sidebar_label: Dialog
+original_id: dialog
+---
+
+## Overview
+
+Dialog は、ダイアログボックスを表示します。
+
+<div class="sample-container" id="dialog">
+  <div id="sample-container__components">
+    <iframe id="iframe" title="dialog image" width="700px" height="300px"></iframe>
```

Comment:
Please double-check that the width and height are suitable for the new script including footer Button.

Reply:
The width and heigh are suitable after adding the Buttons to fotter.
It's confirmed at record.

Thank you!